examples/demo-react-native/scripts/bloatBundle.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
```

Comment:
Perhaps we should add a comment to this file with a link to [the gh issue](https://github.com/wix/Detox/issues/3507) to provide future context of why we have a bloat test

Reply:
Done.